### ISSUE TITLE
Timeshift setting "system" of a backup does not meet current OS version 

### ISSUE THREAD

**Issue by mabraton** [linuxmint/timeshift#363]
**Description**
When I do backups after upgrading from Linux Mint 21.3 to Linux Mint 22, timeshift still says "Linux Mint 21.3 (virginia)" in the system column of the backups description.

**To Reproduce**
See above

**Expected behavior**
It should say "Linux Mint 22 (wilma)" 

**Screenshots**
![Bildschirmfoto vom 2024-10-31 15-32-23](https://github.com/user-attachments/assets/11951478-6cc6-45f8-a84a-0d986e340d43)

**System:**
 - Linux Mint 22 (wilma)
 - Cinnamon 6.04
 - timeshift

**Comment by scott-dft** [comment]
Think this may be a Mint upgrade issue if it is the same omissions I seen when using the Upgrade Tool.
For me the Upgrade Tool failed to change the following files:
- etc/issue
- etc/os-release
- etc/lsb-release

It did update etc/linuxmint/info and etc/upstream-release/lsb-release (for the Ubuntu base).
I had to check a live-usb run of Mint 22 to get the correct entries for the first three files.

I would check those files as I'm guessing Timeshift picks the text to use from one of those ...

**Comment by ygerlach** [comment]
> I would check those files as I'm guessing Timeshift picks the text to use from one of those ...

Yes, timeshift checks for `/etc/lsb-release` if that does not exists, it tries `/etc/os-release`
https://github.com/linuxmint/timeshift/blob/master/src/Utility/LinuxDistro.vala#L60
